specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-03-01/cosmos-db.json

```diff
@@ -5156,6 +5156,13 @@
             "$ref": "#/definitions/VirtualNetworkRule"
           }
         },
+        "privateEndpointConnections": {
+          "type": "array",
+          "description": "List of Private Endpoint Connections configured for the Cosmos DB account.",
+          "items": {
+            "$ref": "../../preview/2019-08-01-preview/privateEndpointConnection.json#/definitions/PrivateEndpointConnection"
```

Comment:
Please move this definition across into this API version (2020-03-01) rather than directly referencing a definition in a different API version.